src/_posts/platform/deployment/2000-01-01-deploy-java-jar-war.md

```diff
@@ -93,22 +93,26 @@ $ scalingo --app my-app env-set JAVA_VERSION=1.9
 
 ### Webapp Runner / Tomcat version selection
 
-By default, Tomcat is installed with the last version of the webapp-runner. If
-you want to use another version, you can defined the environment variable
-`JAVA_WEBAPP_RUNNER_VERSION`. The available versions are:
+By default, the latest `9.0.x` version of Tomcat is installed. If you want to
+use another version, you can define the environment variable
+`JAVA_WEBAPP_RUNNER_VERSION`, like so:
 
-- `8.5.11.3`
-- `8.5.51.0`
-- `9.0.52.1`
-- `9.0.68.1`
-- `9.0.80.0`
-- `9.0.83.0` (default)
+```sh
+scalingo --app my-app env-set JAVA_WEBAPP_RUNNER_VERSION=10.1.16.0
+```
 
-The 8.5.x versions are installing Tomcat 8, and the 9.0.x releases are
-installing Tomcat 9.0:
+The latest available versions currently are:
 
-```sh
-# Install Tomcat 8
+| Tomcat Version | `scalingo-20`     | `scalingo-22`     |
+| -------------: | ----------------: | ----------------: |
+| `8.5`          | up to `8.5.51.0`  | up to `8.5.51.0`  |
+| **`9.0`**      | up to `9.0.83.1`  | up to `9.0.83.1`  |
+| **`10.1`**     | up to `10.1.16.0` | up to `10.1.16.0` |
```

Comment:
Do we want to have a column for each stack versions? They have exactly the same supported versions. And from what I remember we have no code making the distinction between the stacks.

Reply:
I added it to remain homogeneous with the other versions table we have (PHP, nginx,...).
Removed in https://github.com/Scalingo/documentation/pull/2437/commits/459272c94f9cbdfc8920f1a1177105e2817f8bee